soweego/linker/classify.py

```diff
@@ -122,14 +128,32 @@ def _zero_when_different_names(prediction, wikidata, target):
     return 0.0 if wd_names.isdisjoint(target_names) else prediction[0]
 
 
-def _add_missing_feature_columns(classifier, feature_vectors):
+def _one_when_wikidata_link_correct(prediction, target):
+    qid, tid = prediction.name
+
+    if target.get(constants.URL) is not None:
```

Comment:
Probably this check should be made **before** firing the `apply`, i.e., where the `apply` is called.